WeatherApp/WeatherApp.Webpage/Controllers/ObservationController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WeatherApp.WebSite.Models;
+
+namespace WeatherApp.WebSite.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class ObservationController : ControllerBase
+    {
+        readonly IObservationRepository _observationRepository;
+
+        public ObservationController(IObservationRepository observationRepository)
+        {
+            _observationRepository = observationRepository;
+        }
+
+        public object Observation { get; private set; }

```

Comment:
It is unused. Can be deleted. Anyway, why its type is `object` and not `Observation`?

Reply:
Thank you for your remark, it is fixed now.